examples/r-job/README.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Create a job using R"
+weight: 2
+description: "Setting and executing a job on Saturn Cloud using R"
+---
+
+
+A Saturn Cloud job is a computing environment set up to run recurring tasks. You can start a job in 3 ways:
+
+1. on a schedule 
+2. manually by pressing start button 
+3. via an API 
+
```

Comment:
You should first have a section that is like "setting up the Saturn Cloud job resource" take briefly talks about this will be made into a job later